config/vufind/Demo.ini

```diff
@@ -236,6 +236,16 @@ extraHoldFields = "comments:requestGroup:pickUpLocation:proxiedUsers:startDate:r
 ; Should we include a link to a fake OPAC?
 fakeOpacLink = false
 
+; Block ILS methods during given times. Format is "method[setting] = value"
+; method  - Method in ILS driver
+; setting - Valid settings are startDate, endDate, recurringStart, recurringEnd
```

Comment:
Maybe it's worth adding a little more explanation, like:

startDate and endDate specify fixed start and end dates (with optional times) to make the method unavailable. You do not need to specify both -- you can use either on its own for an open-ended range.

recurringStart and recurringEnd specify a time window that will block the method every day. If recurringEnd is lower than recurringStart, the driver will assume a time window crossing from one day into the next day. When setting a recurring window, both recurringStart and recurringEnd MUST be set.

If startDate or endDate is specified, the recurring settings will be ignored; you cannot combine these two types of settings.